nodebuilder/p2p/host.go

```diff
@@ -41,14 +41,16 @@ func routedHost(base HostBase, r routing.PeerRouting) hst.Host {
 
 // host returns constructor for Host.
 func host(params hostParams) (HostBase, error) {
+	buildInfo := node.GetBuildInfo()
+	userAgent := fmt.Sprintf("%s/%s", params.Net, buildInfo.SemanticVersion)
```

Comment:
Let's make it so it has a form of `celestia-node/Arabica-10/v0.12/abc32das`

Reply:
Hi @Wondertan, I need a little help. How can I get the network alias (like arabica, mocha, etc.) from the network.go file to host.go?

Reply:
It's already passed through via params.Net